composeApp/src/desktopMain/kotlin/edu/dyds/movies/domain/usecase/GetMovieDetailsUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package edu.dyds.movies.domain.usecase
+
+import edu.dyds.movies.domain.entity.Movie
+import edu.dyds.movies.domain.repository.MoviesRepository
+
+interface GetMovieDetailsUseCase {
+    suspend fun getMovieDetails(id: Int): Movie?
```

Comment:
para evitar la repetición
```suggestion
    suspend fun execute(id: Int): Movie?
```